### PR TITLE
[ARRDT] Accept Relay fetch policies instead

### PR DESCRIPTION
**Pull request by alloy**

### CODE REVIEW

File: packages/apollo-react-relay-duct-tape/src/convertFetchPolicy.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import type { WatchQueryFetchPolicy as ApolloWatchQueryFetchPolicy } from "@apollo/client";
+import type { FetchPolicy } from "./types";
+
+/**
+ * This is for internal use only.
+ */
+type PrivateFetchPolicy = FetchPolicy | "no-cache";
```
Comment (alloy) [inline packages/apollo-react-relay-duct-tape/src/convertFetchPolicy.ts:7]: https://github.com/microsoft/graphitation/blob/fe9b9dab034befb7cee2ae8141f3f49dc3237999/packages/apollo-react-relay-duct-tape/src/storeObservation/compiledHooks/useCompiledPaginationFragment.ts#L81

File: packages/apollo-react-relay-duct-tape/src/storeObservation/compiledHooks/useCompiledPaginationFragment.ts [hunk 2]
```diff
@@ -128,7 +128,10 @@ function useLoadMore({
       // TODO: Measure if invoking `refetch` leads to React updates and if it
       //       makes sense to wrap it and the following setIsLoadingMore(true)
       //       call in a batchedUpdates callback.
-      const disposable = refetch(newVariables, refetchOptions);
+      const disposable = refetch(
+        newVariables,
+        refetchOptions as RefetchOptions
```
Comment (vladar) [inline packages/apollo-react-relay-duct-tape/src/storeObservation/compiledHooks/useCompiledPaginationFragment.ts:133]: Is this casting unavoidable?

**Review by vladar — APPROVED** [review]
Looks good to me. One comment but not blocking this PR.

File: packages/apollo-react-relay-duct-tape/src/storeObservation/compiledHooks/useCompiledPaginationFragment.ts (hunk 2 above)
Comment (alloy) [reply packages/apollo-react-relay-duct-tape/src/storeObservation/compiledHooks/useCompiledPaginationFragment.ts:133]: Perhaps if the refetching was extracted from useRefetchableFragment itself and shared between the two hooks. I’ll mull it over when I have some refactor time, but for now this is a fine-iSH private detail imo.

**Pull request merged**